resources/views/tournaments/view.blade.php

```diff
@@ -256,6 +256,57 @@
                     </div>
                 </div>
             @endif
+            {{--Videos--}}
+            <div class="bracket">
+            @if ($tournament->concluded)
+                <h5>
+                    <i class="fa fa-video-camera" aria-hidden="true"></i>
+                    Videos
+                </h5>
+                {{--Add video--}}
+                <button class="btn btn-primary btn-xs" id="button-add-videos"
+                        onclick="toggleVideoAdd(true)">
+                    <i class="fa fa-video-camera" aria-hidden="true"></i> Add videos
+                </button>
+                <button class="btn btn-primary btn-xs hidden-xs-up" id="button-done-videos"
+                        onclick="toggleVideoAdd(false)">
+                    <i class="fa fa-check" aria-hidden="true"></i> Done
+                </button>
+                <div id="section-add-videos" class="hidden-xs-up small-text">
+                    <hr/>
+                    <div class="p-b-1">
+                        Add a Youtube video
+                    </div>
+                    {!! Form::open(['method' => 'POST', 'url' => "/videos",
+                        'class' => 'form-inline']) !!}
+                        {!! Form::hidden('tournament_id', $tournament->id) !!}
+                        <div class="form-group">
+                            {!! Form::label('video_id', 'Youtube Video ID') !!}
```

Comment:
It would be nicer if this field accepted Youtube URLs and Youtube video IDs as well. More easier from the user. You can decide if it's an URL or ID on the server side and extract the ID if necessary. Be sure to use the "v" GET parameter only for URLs.